app/js/index.js

```diff
@@ -6,6 +6,12 @@ import Event from './components/Event';
 
 const {Route, RouteHandler} = Router;
 
+//Have to manually remove auto-added fb hash
+//see here: https://github.com/jaredhanson/passport-facebook/issues/12#issuecomment-5913711
+if(window.location.hash && window.location.hash === "#_=_" || window.location.hash === "_=_"){
```

Comment:
React router is currently using hashes for URLs (something like `www.example.com/#/` to match root).  I'm not sure if the extra hashes that facebook appends would affect it

-----Original Message-----
From: "dani5447" <notifications@github.com>
Sent: ‎10/‎3/‎2015 12:00
To: "bbesmanoff/indoor-umbrellas" <indoor-umbrellas@noreply.github.com>
Cc: "Brian Besmanoff" <bbesmanoff@gmail.com>
Subject: Re: [indoor-umbrellas] Fixes #31 page redirects (#37)

In app/js/index.js:
> @@ -6,6 +6,12 @@ import Event from './components/Event';
>  
>  const {Route, RouteHandler} = Router;
>  
> +//Have to manually remove auto-added fb hash
> +//see here: https://github.com/jaredhanson/passport-facebook/issues/12#issuecomment-5913711
> +if(window.location.hash && window.location.hash === "#_=_" || window.location.hash === "_=_"){
Not sure what you mean by that? I tried capturing and redirecting in the server js but that didn't work.
—
Reply to this email directly or view it on GitHub.